run-tests.php

```diff
@@ -2075,10 +2069,10 @@ function run_test($php, $file, $env)
 					$start = $end = $length;
 				}
 				// quote a non re portion of the string
-				$temp = $temp . preg_quote(substr($wanted_re, $startOffset, ($start - $startOffset)),  '/');
+				$temp .= preg_quote(substr($wanted_re, $startOffset, $start - $startOffset),  '/');
```

Comment:
@ntzm How did you detect this? Some PHPCS rule? :smile:

Reply:
PHPStorm inspection :D